src/Compiler/NodeVisitor/ChangeFiltersNodeVisitor.php

```diff
@@ -93,12 +93,12 @@ private function createFilterCallNode(string $filterName, array $args): ?Node
             ], $args);
         }
 
-        if (is_string($filter)) {
-            return new FuncCall(new FullyQualified($filter), $args);
+        if ($filter instanceof Closure || (is_string($filter) && str_starts_with($filter, 'Closure('))) {
```

Comment:
I would add `callable(` and `\Closure(` to allowed prefixes. Maybe make it part of some helper?

Reply:
added callable( prefix with tests

Reply:
oh and now I noticed \Closure... I'll add